log-viewer-webui does not build.

### Bug

The log-viewer-webui component does not build, causing the entire build to fail:

```
> log-viewer-webui-client@0.1.0 build
> tsc -b && vite build --emptyOutDir --outDir /home/junhao/workspace/clp/build/log-viewer-webui/client

src/AntdApp.tsx:1:1 - error TS6133: 'React' is declared but its value is never read.

1 import React from "react";
  ~~~~~~~~~~~~~~~~~~~~~~~~~~

src/ui/MainLayout.tsx:1:8 - error TS6133: 'React' is declared but its value is never read.

1 import React, {useState} from "react";
         ~~~~~


Found 2 errors.

exit status 2
```

### CLP version

d5726d0c0164385a152286d3b95e0252cc3c0b19

### Environment

```
junhao@GIGABYTE:~/workspace/clp$ cat /etc/os-release 
PRETTY_NAME="Ubuntu 22.04.3 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.3 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
```

### Reproduction steps

1. Clone the repo.
2. Run `task`.
3. Observe build failure and error logs are shown as above.

--- a/components/log-viewer-webui/client/src/AntdApp.tsx
+++ b/components/log-viewer-webui/client/src/AntdApp.tsx
@@ -1,5 +1,3 @@
-import React from "react";
-
 import MainLayout from "./ui/MainLayout";
 
 

--- a/components/log-viewer-webui/client/src/ui/MainLayout.tsx
+++ b/components/log-viewer-webui/client/src/ui/MainLayout.tsx
@@ -1,4 +1,4 @@
-import React, {useState} from "react";
+import {useState} from "react";
 
 import {
     SearchOutlined,

